x-pack/test/functional/services/search_sessions.ts

```diff
@@ -138,21 +138,31 @@ export class SearchSessionsService extends FtrService {
     this.log.debug('Deleting created search sessions');
     // ignores 409 errs and keeps retrying
     await this.retry.tryForTime(10000, async () => {
-      const { body } = await this.supertest
+      const { body } = await this.security.testUserSupertest
         .post('/internal/session/_find')
         .set('kbn-xsrf', 'anything')
         .set('kbn-system-request', 'true')
-        .send({ page: 1, perPage: 10000, sortField: 'created', sortOrder: 'asc' })
+        .send({
+          page: 1,
+          perPage: 10000,
+          sortField: 'created',
+          sortOrder: 'asc',
+        })
         .expect(200);
 
       const { saved_objects: savedObjects } = body as SavedObjectsFindResponse;
-      if (savedObjects.length) {
+
```

Comment:
@Dosant should this be called saved objects?  
I ask because I thought these were not "officially" saved objects.
I mean, they kinda are, but they use a different ui view, and they are not import / exportable iirc.

Yeah names are hard and this is mostly a **nit**, but it does "fee" confusing to me.

Reply:
I agree.
It is like this because we simply proxied the savedobject.find method :( 
this is also an internal API so we didn't think about the semantics